lib/main.js

```diff
@@ -118,6 +118,12 @@ function logServerPort (address, port) {
   logger.info(logMessage);
 }
 
+function initHeapdump (args) {
+  if (args.isHeapdumpEnabled) {
+    require('heapdump');
```

Comment:
Does this not need some sort of call, somewhere, to `heapdump.writeSnapshot`? Or is requiring it somehow enough?